test/qa_thread_affinity.cpp

```diff
@@ -0,0 +1,161 @@
+#include <boost/ut.hpp>
```

Comment:
the qa_thread_affinity unit-test should be added to the CI/CMakeList.txt.

Reply:
thanks for catching that, should be fixed.